dotnet/samples/Concepts/AgentSyntax/Getting_Started/Step5_JsonResult.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Examples;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Agents;
+using Microsoft.SemanticKernel.Agents.Chat;
+using Microsoft.SemanticKernel.ChatCompletion;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace GettingStarted;
+
+/// <summary>
+/// Demonstrate parsing JSON response using <see cref="JsonResultTranslator"/>.
+/// </summary>
+public class Step5_JsonResult(ITestOutputHelper output) : BaseTest(output)
+{
+    private const string TutorName = "Tutor";
+    private const string TutorInstructions =
+        """
+        Think step-by-step and rate the user input on creativity and expressivness from 1-100.
+
+        Respond in JSON format.
+        
+        Leave JSON properties empty if no associated information is available.
+                
+        The JSON schema can include only:
```

Comment:
`JSON response can include`

Would it be better to have

```
Use the following JSON schema

<Add a schema here>
```

Reply:
Updated